modules/ROOT/pages/backup-restore/inspect.adoc

```diff
@@ -22,6 +22,7 @@ The metadata contains the following information:
 * *Compressed*: indicates whether the backup data inside the backup file is compressed.
 * *Lowest transaction ID*: when the backup is full, this value is always 1, and when it is a differential backup, the value corresponds to the first transaction ID the backup starts with.
 * *Highest transaction ID*: similarly, this value indicates the last transaction ID stored in the backup file.
+* label:new[Introduced in 2025.06] *Recovered*: shows whether a database backup is recovered or not.
```

Comment:
@sumyiren, hey! I think everything looks good. One question: Do we need to explain the meaning of this column?